allennlp/training/trainer.py

```diff
@@ -213,7 +215,6 @@ def _train_epoch(self, epoch: int) -> dict:
         self._last_log = time.time()
         batch_num = 0
 
-        logger.info("Training")
```

Comment:
I would keep this here, as we have progress bars for training and validation separately, and this makes it more obvious which one is which.

Reply:
I'll revert this commit--it wasn't supposed to be in the CR.